File: PRAK404/PRAK404-2210817310013-Ryan Muhammad Irfan.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. EXIT")

    x = int(input("Masukan Pilihan : "))
    if x == 5:
        print("Terimakasih, telah menggunakan kalkulator RYANMUHAMMADIRFAN")
        break
    elif x <= 0 or x >= 6:
        print("Input anda salah, silahkan coba lagi\n")
        continue

    y = float(input("Masukan nilai pertama : "))
    z = float(input("Masukan nilai kedua : "))

    if x == 1:
        h = y + z
        print(f"Hasil perkalian antara {format(y, '.2f')} dengan {format(z, '.2f')} adalah {format(h, '.2f')}\n")

    elif x == 2:
        h = y - z
        print(f"Hasil perkalian antara {format(y, '.2f')} dengan {format(z, '.2f')} adalah {format(h, '.2f')}\n")

    elif x == 3:
        h = y * z
        print(f"Hasil perkalian antara {format(y, '.2f')} dengan {format(z, '.2f')} adalah {format(h, '.2f')}\n")

    elif x == 4:
        h = y / z
        print(f"Hasil perkalian antara {format(y, '.2f')} dengan {format(z, '.2f')} adalah {format(h, '.2f')}\n")

File: PRAK404/PRAK404-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>

int main()
{
    int x;
    float y, z, h;

    while (x != 5)
    {
        printf("Pilih program\n");
        printf("1.Penjumlahan\n");
        printf("2.Pengurangan\n");
        printf("3.Perkalian\n");
        printf("4.Pembagian\n");
        printf("5.Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &x);
        if (x == 5)
        {
            printf("Terimakasih, telah menggunakan kalkulator RYANMUHAMMADIRFAN");
        }
        else if (x <= 0 || x >= 5)
        {
            printf("Input anda salah, silahkan coba lagi\n\n");
        }
        else
        {
            printf("Masukkan Nilai pertama : ");
            scanf("%f", &y);
            printf("Masukkan Nilai kedua : ");
            scanf("%f", &z);
            if (x == 1)
            {
                h = y + z;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", y, z, h);
            }
            else if (x == 2)
            {
                h = y - z;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", y, z, h);
            }
            else if (x == 3)
            {
                h = y * z;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", y, z, h);
            }
            else
            {
                h = y / z;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", y, z, h);
            }
        }
    }
}